dist/tools/prep/prep.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
```

Comment:
I would actually keep it version agnostic and just use `python` instead of `python3` (here and all following files)